recipes/libffi/0001-libffi-Don-t-be-smart-about-toolexeclibdir.patch

```diff
@@ -1,17 +1,17 @@
-From 9d4196a8868abd07dfb1ce9a3fb1dda23872061e Mon Sep 17 00:00:00 2001
+From 21b2c114d58f7e305a4c26bf761dd2496bdcf18e Mon Sep 17 00:00:00 2001
 From: Edward Hervey <bilboed@bilboed.com>
 Date: Thu, 29 Jan 2015 15:37:02 +0100
 Subject: [PATCH] libffi: Don't be smart about toolexeclibdir
```

Comment:
Mostly so we remember to keep it when we move back to upstream. Upstream tries to be smart about libdir and uses `lib64` instead of `lib`, which is wrong. It's a no-op here, but it won't be for version >3.2.